packages/core/src/components/select/_select.scss

```diff
@@ -140,5 +147,22 @@
         color: rgba($ray-color-gray-60, 0.4);
       }
     }
+
+    &--simple {
```

Comment:
new addition to `ray-select` to support things like the flag dropdown

<img width="218" alt="Screen Shot 2019-05-10 at 10 34 27 AM" src="https://user-images.githubusercontent.com/49030/57534964-34db4600-730f-11e9-8b94-2e6df9607a09.png">